src/parser/Parser.ts

```diff
@@ -618,6 +618,9 @@ export class Parser {
                 then = advance();
             }
 
+            //consume any trailing colons
+            while (check(Lexeme.Colon)) { advance(); }
```

Comment:
This might need to get moved into the `if(match(Lexeme.Newline))` on line 624 - I suspect using `if (condition):` forces the parser into "multi-line `if`" mode.  I still have to confirm against RBI but I'd bet this isn't valid syntax:

```brightscript
function t()
    if 1 < 2: return true

    return false
end function
```

Reply:
Confirmed that a `:` after the condition causes the compiler to assume it's a multi-line if statement :/